src/collective/cover/tests/test_collection_tile.py

```diff
@@ -76,6 +77,16 @@ def test_populate_tile_with_object_string(self):
         self.assertTrue(self.tile.data.get('footer'))
         self.assertEqual(self.tile.data.get('uuid'), IUUID(obj))
 
+    def test_populate_tile_without_image(self):
```

Comment:
@rodfersou this test does not work: you have to create a test in collective.nitf for this, make the test fail and test with this branch to see if that fixes the issue.

rendering the tile must rise an exception and this is not happening here,